client/my-sites/customer-home/main.jsx

```diff
@@ -295,6 +276,22 @@ class Home extends Component {
 						</div>
 					) }
 				</div>
+				{ isRecentlyMigratedSite && (
+					<Card className="customer-home__migrate-card" highlight="info">
+						<img
+							src="/calypso/images/illustrations/fireworks.svg"
```

Comment:
https://github.com/Automattic/wp-calypso/pull/38356 migrated this to use webpack file loader. The static image was temporarily preserved to prevent 404s, but please avoid it in the future.

Fix in #39861